sportsdataverse/mlb/retrosheet.py

```
@@ -43,36 +44,36 @@ def retrosheet_ejections() -> pd.DataFrame():
         None
     
     Returns:
-        A pandas Dataframe with the biographical information of known MLB ejections.
+        A pandas Dataframe with the ejection data of known MLB ejections.
     """
-    people_url = "https://www.retrosheet.org/Ejecdata.txt"
-    resp = download(people_url)
-    resp_str = str(resp, 'UTF-8')
-    resp_str = StringIO(str(resp, 'UTF-8'))
-    ejections_df = pd.read_csv(resp_str,sep=",")
-    ejections_df.columns = ['game_id','date','dh','ejectee','ejectee_name','team','job','umpire','umpire_name','inning','reason']
-    return ejections_df
-
+    ejections_url = "https://www.retrosheet.org/Ejecdata.txt"
+    try:
+        ejections_df = pd.read_csv(ejections_url,sep=",")
+        ejections_df.columns = ['game_id','date','dh','ejectee_id','ejectee_name','team','job','umpire_id','umpire_name','inning','reason']
+        return ejections_df
+    except:
+        print('Could not downoad the MLB ejection file from Retrosheet. Check your internet connection before running this function agian.')
```

same thing re: spelling and swallowing errors